src/index.ts

```diff
@@ -12,7 +12,7 @@ function isRelativeUrlWithoutProtocol(url: string): boolean {
 
 // adapted from https://stackoverflow.com/a/29824550/2601552
 function decodeHtmlCharacters(str: string) {
-  str = str.replace(htmlTabEntityRegex, "&#9;");
+  str = str.replace(htmlSpaceEntityRegex, "&#9;");
```

Comment:
It's a little odd that `&tab;` and `&newline;` both get converted to `&#9;`, which is a tab. The `&#9;` then gets decoded and stripped out as a control character. Since we know `&tab;` and `&newline;` are both control characters, can we just eliminate them here?
```js
  str = str.replace(htmlSpaceEntityRegex, "");
```
That does change the nature of what `decodeHtmlCharacters` does, though, but decoding `&newline;` as a tab is already doing that.

Reply:
yeah - i like that a lot better (eliminating them entirely). fixed!